crypto/jwx.go

```diff
@@ -338,6 +434,35 @@ func SignatureAlgorithm(key crypto.PublicKey) (jwa.SignatureAlgorithm, error) {
 	}
 }
 
+func EncryptionAlgorithm(key crypto.PublicKey) (jwa.KeyEncryptionAlgorithm, error) {
```

Comment:
I think this doesn't need to be exported

Reply:
Agree.